packages/server/src/routes/grants.js

```diff
@@ -235,8 +235,15 @@ router.delete('/:grantId/interested/:agencyId', requireUser, async (req, res) =>
     const { user } = req.session;
     const { grantId, agencyId } = req.params;
     const { agencyIds } = req.body;
-    console.log(agencyIds);
-    const allowed = await isUserAuthorized(user, agencyIds || agencyId);
+    let authorizedAgencyId = agencyId;
+    if (agencyIds.length > 0) {
+        // Note: ran into a ES lint error so had to declare the variable first
+        // and then assign it to authorizedAgencyId.
+        const [firstAgency] = agencyIds;
```

Comment:
I know it's "temporary" but I don't really like this idea of just using the first agency; doing so basically negates the whole permission check since someone could pass one valid/authorized agency, followed by anything else